src/AstTranslator.cpp

```diff
@@ -306,10 +306,11 @@ std::unique_ptr<RamCondition> AstTranslator::translateConstraint(
     class ConstraintTranslator : public AstVisitor<std::unique_ptr<RamCondition>> {
         AstTranslator& translator;
         const ValueIndex& index;
+        const AuxiliaryArity* aaa;
 
     public:
-        ConstraintTranslator(AstTranslator& translator, const ValueIndex& index)
-                : translator(translator), index(index) {}
+        ConstraintTranslator(AstTranslator& translator, const ValueIndex& index, const AuxiliaryArity* aaa)
```

Comment:
I do really like acronyms, but a more descriptive variable name would be better, such as `auxiliaryArityAnalysis`

Reply:
Let's do auxArityAnalysis as a compromise.